#!/usr/bin/python3
# 6-raise_exception_msg

def raise_exception_msg(message=""):
    raise NameError(message)
